js/build.js

```diff
@@ -10,17 +10,13 @@ Fliplet.Widget.instance('login', function(data) {
     authProcessing: 'Verifying...',
     sendDefault: 'Send new code',
     sendProcessing: 'Sending...',
-    continuDefault :'Continue',
+    continueDefault: 'Continue',
     continueProcessing: 'Please wait...'
   };
+
   _this.$container = $(this);
   _this.data = data;
   _this.pvNameStorage = 'fliplet_login_component';
-  var studioUrls = {
```

Comment:
I'm guessing these aren't required anymore?

Reply:
That's right. I don't see any reference of it.